### PR TITLE
Update release notes draft to version v1.21.0-beta.0

### PR DESCRIPTION
**Pull request by sladyn98**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.21.0-beta.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by wilsonehusin** [comment]
awesome, thanks for getting this out @sladyn98! a couple of things before we get this done:
- we need the `release-notes-draft.{md,json}` to be in `release-notes/` directory -- we're mid-migration, should be done soon! roughly doing `mv release-notes-draft.* release-notes/` in `release-1.21` directory should do it
- after moving them, you'll need to restore the old `release-notes-draft.md` by `git checkout master -- release-notes-draft.md` in the same directory
- to validate if you've done the right thing, you can run `git diff master --name-only` and you should see all the files being listed are under `releases/release-1.21/release-notes` directory

/assign
/cc ashnehete melodychn pmmalinov01 soniasingla 
/uncc cartyc thejoycekung

**Comment by pmmalinov01** [comment]
Just moved the files Wilson pointed. Great work.

**Comment by sladyn98** [comment]
@wilsonehusin do I have to move these files after every PR i make ? or is it just a one time thing

**Comment by sladyn98** [comment]
> Just moved the files Wilson pointed. Great work.
> /lgtm

Will I have to do this locally or like have you done it on my branch and I can pull them in ?

**Review by soniasingla — COMMENTED** [review]
@sladyn98 amazing work! 

You don't need to do it for each single PR, it should be a one time thing! Also, if it gets too messy, feel free to abandon this PR and raise a new PR with suggested changes as mentioned by @wilsonehusin above :)

If there are more questions, feel free to ask here or on slack `#release-notes`

**Comment by wilsonehusin** [comment]
sorry to clarify -- for now we need to do this on every PR manually, until https://github.com/kubernetes/release/pull/1922 gets merged 😅

**Comment by sladyn98** [comment]
@wilsonehusin I moved all the files according to your steps. Do take a look and let me know

File: releases/release-1.21/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -28,70 +29,98 @@
...
 - Adds the ability to pass --strict-transport-security-directives to the kube-apiserver to set the HSTS header appropriately.  Be sure you understand the consequences to browsers before setting this field. ([#96502](https://github.com/kubernetes/kubernetes/pull/96502), [@249043822](https://github.com/249043822)) [SIG Auth]
 - Base-images: Update to debian-iptables:buster-v1.4.0
   - Uses iptables 1.8.5
   - base-images: Update to debian-base:buster-v1.3.0
   - cluster/images/etcd: Build etcd:3.4.13-2 image
     - Uses debian-base:buster-v1.3.0 ([#98401](https://github.com/kubernetes/kubernetes/pull/98401), [@pacoxu](https://github.com/pacoxu)) [SIG Testing]
 - Export `NewDebuggingRoundTripper` function and `DebugLevel` options in the k8s.io/client-go/transport package. ([#98324](https://github.com/kubernetes/kubernetes/pull/98324), [@atosatto](https://github.com/atosatto))
+- Feature gate RootCAConfigMap is graduated to GA in 1.21 and will be removed in 1.22. ([#98033](https://github.com/kubernetes/kubernetes/pull/98033), [@zshihang](https://github.com/zshihang)) [SIG API Machinery and Auth]
```
Comment (wilsonehusin) [inline releases/release-1.21/release-notes/release-notes-draft.md:72]: We should reword this with "unconditionally enabled" like other GA features. For future PRs though, doesn't have to be now.

**Review by wilsonehusin — APPROVED** [review]
First pass looks good -- thank you for your contribution and congratulations on your first PR!

/lgtm
/approve

**Pull request merged**